backend/main/server/app.go

```diff
@@ -193,6 +193,7 @@ func (a *App) Initialize() {
 func (a *App) Run() {
 	addr := fmt.Sprintf(":%s", os.Getenv("API_PORT"))
 	log.Info().Msgf("Starting server on %s ...", addr)
+	go shared.StartJobWorkers()
```

Comment:
there is a quirk here where a SIGINT is consumed by the worker and the main server remains running, looking for a solution on how to couple the listeners to close and open together.  Researching more into go routine channels for this, pairing is welcomed